scrapers/govScrapers/getColombia.js

```diff
@@ -37,7 +47,7 @@ const colombiaData = async () => {
 		return {
 			updated: Date.now(),
 			departments: transformData(colombiaResponse.data, false),
-			cities: transformData(colombiaResponse.data, true)
```

Comment:
Don’t call the new field `municipalities`, that would break code that someone may have written to use the `cities`field.

Reply:
You are right, can you please check again? thanks